test/NeoSharp.Core.Test/SmartContracts/UtContractParameterType.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System.Numerics;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NeoSharp.Core.Cryptography;
+using NeoSharp.Core.SmartContract.ContractParameters;
+using NeoSharp.Core.Types;
+using NeoSharp.VM;
+
+namespace NeoSharp.Core.Test.SmartContracts
+{
+    [TestClass]
+    public class UtContractParameterTypeTests
+    {
+        private ScriptBuilder scriptBuilder;
+
+        [TestInitialize]
+        public void SetUp(){
+            scriptBuilder = new ScriptBuilder();       
```

Comment:
This is a unit test so ScriptBuilder should be mocked and we just need to ensure that the value is being passed correctly into ScriptBuilder (verify that emitPush is called with the correct value and type).